core/modules/filter/filter.admin.inc

```diff
@@ -852,6 +853,14 @@ function filter_editor_image_upload_settings_form($format) {
     $form['scheme'][$scheme]['#attributes']['data-label'] = t('Storage: @name', array('@name' => $info['name']));
   }
 
+  $form['alt_required'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('<em>Alt</em> text required'),
+    '#description' => t('Requiring alternative text is strongly encouraged to improve accessibility for all users.'),
```

Comment:
"users" here should be "people". 

Or, for simplicity in all languages, leave it at "Requiring alternative text is strongly encouraged to improve accessibility."